### PR TITLE
[READY] Darkmode fixes

### PR DESCRIPTION
**Pull request by actioninja**
## About The Pull Request
main window + channel colors:
image
descriptions:
image
deadchat:
image
Syndicate and AI private:
image
Clown megaphone:
image
Admin tickets (not accurate I need to update screenshot):
image
Cult:
image

There's more changes but I'm too lazy to go through and screenshot every single one. Just know that everything is readable now.

Changes the darkmode colors to match tgui and retunes a lot of the text colors to be more readable.

## Why It's Good For The Game

- Consistency
- Much easier to read text
- Nicer looking goonchat buttons
- Higher contrast

## Changelog
:cl:
fix: admin pms display in the proper colors
fix: some uis are no longer unreadable.
tweak: Updated the colors of the dark mode ui.
tweak: darkmode is no longer the default
/:cl:

### CODE REVIEW

**Comment by Kmc2000** [comment]
I'd say this sorts most of the issues.
Does this address the weird ooc colours?

**Comment by actioninja** [comment]
> I'd say this sorts most of the issues.
> Does this address the weird ooc colours?

Not yet, OOC color is controlled internally and not actually by stylesheets. I need to write some special code to be able to have different darkmode and lightmode OOC.

**Comment by actioninja** [comment]
> FIX deadchat and death notices

I did actually, let me add some more screenshots of tweaked colors to the main post.

**Comment by mrhugo13** [comment]
anything for darkmode UI yeEEEEEEEEEEEEEEEEEEEEE

**Comment by Barhandar** [comment]
Fix the different input-area font too, possibly separately, pls.

**Comment by Dorsisdwarf** [comment]
Why does this fuck with admin colours _more_?

**Comment by actioninja** [comment]
> Why does this fuck with admin colours _more_?

I just fixed it, there was a typo in a bunch of shit.

File: code/modules/goonchat/browserassets/js/browserOutput.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ var opts = {
 	'wasd': false, //Is the user in wasd mode?
 	'priorChatHeight': 0, //Thing for height-resizing detection
 	'restarting': false, //Is the round restarting?
-	'darkmode':true, //Are we using darkmode? If not WHY ARE YOU LIVING IN 2009???
+	'darkmode':false, //Are we using darkmode? If not WHY ARE YOU LIVING IN 2009???
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/goonchat/browserassets/js/browserOutput.js]: revert

Comment (Kmc2000) [reply code/modules/goonchat/browserassets/js/browserOutput.js]: REEEEEEE

Comment (ExcessiveUseOfCobblestone) [reply code/modules/goonchat/browserassets/js/browserOutput.js]: no rest for the wicked

Comment (ExcessiveUseOfCobblestone) [reply code/modules/goonchat/browserassets/js/browserOutput.js]:
```suggestion
	'darkmode':true, //Are we using darkmode? If not WHY ARE YOU LIVING IN 2009???
```

Comment (PKPenguin321) [reply code/modules/goonchat/browserassets/js/browserOutput.js]: This is not consentual cobby

Comment (Kmc2000) [reply code/modules/goonchat/browserassets/js/browserOutput.js]: God why just end the hate already!!!

Comment (ExcessiveUseOfCobblestone) [reply code/modules/goonchat/browserassets/js/browserOutput.js]: ヽ(°◇° )ノ

**Pull request merged**